params/config.go

```diff
@@ -92,6 +95,12 @@ type WhisperConfig struct {
 	// (if no account file selected, then this password is used for symmetric encryption).
 	Password string
 
+	// AsymKeyFile is a file with an asymmetric key to decrypt messages sent to MailServer.
```

Comment:
If they're mail server related config options shouldn't we prefix them with MailServer like [MailServerRateLimit ](https://github.com/status-im/status-go/blob/2b4a7dfb1c56b02783a884c43af98677361dc3fc/params/config.go#L118) or [MailServerCleanupPeriod](https://github.com/status-im/status-go/blob/2b4a7dfb1c56b02783a884c43af98677361dc3fc/params/config.go#L121)